src/findRange.ts

```diff
@@ -1,5 +1,5 @@
 export function findRange(
-  entries: [timestamp: bigint, _: unknown, __: unknown][],
+  entries: [timestamp: bigint, _: unknown, _: unknown][],
```

Comment:
maybe you can remove this stuff by doing something like `entries: [timestamp: bigint, ...unknown[]]` ?

https://www.typescriptlang.org/play/?#code/C4TwDgpgBAKlC8UDaAjAlgczQO2AGigDpiBXbAa2wHsB3bJAXQYChmAzMgY2DSuymAQAzsAAUAQwBOkgFywAlFADezKGqic+IqAA8EUKZKQAGBgG5V6gPRX1UAHoB+S2s3ZtIfYaQBGcy6gbOydmAF8gA